pkg/integration/tests/interactive_rebase/drop_with_custom_comment_char.go

```diff
@@ -0,0 +1,38 @@
+package interactive_rebase
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var DropWithCustomCommentChar = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Drops a commit with the 'core.commentChar' option set to a custom character",
+	ExtraCmdArgs: []string{},
+	Skip:         false,
+	SetupConfig:  func(config *config.AppConfig) {},
+	SetupRepo: func(shell *Shell) {
+		shell.SetConfig("core.commentChar", ";")
+		shell.CreateNCommits(2)
+	},
+	Run: func(t *TestDriver, keys config.KeybindingConfig) {
+		confirmCommitDrop := func() {
+			t.ExpectPopup().Confirmation().
+				Title(Equals("Delete commit")).
+				Content(Equals("Are you sure you want to delete this commit?")).
+				Confirm()
+		}
```

Comment:
I'm not a fan of having local functions like this in tests. It makes the flow of what happens in which order harder to understand, so I'd just have inlined this below where you need it.

However, I see that you copied this from other existing tests that do it in the same way. How about extracting this to a new file in `pkg/integration/tests/shared`? Doing this for the existing tests could be a separate commit at the beginning of the branch.

Reply:
I'm okay with local functions if they remove lots of repetition (e.g. if you need to drop 5 commits in the one test), but I agree about erring on the side of inlining everything.

I'm also fine with this being a shared function. You can chuck it in `pkg/integration/components/common.go` if you like

Reply:
I felt the function was simple enough, so I inlined it.